doc/src/create_atoms.rst

```diff
@@ -395,6 +396,12 @@ correct number of particles are inserted, in a perfectly random
 fashion.  Which lattice sites are selected will change with the number
 of processors used.
 
+.. versionadded:: TBD
+
+The *group* keyword adds the newly created atoms to the named
+:doc:`group <group>`.  If the group does not yet exist it will be
+created.  All created atoms are always added to the group "all".
```

Comment:
Maybe worth specifying that only the last group keyword will be used? I.e. one cannot list two groups and expect atoms to be added to both.

Reply:
Good point.